oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -508,6 +508,21 @@ Netty-http:
 c3p0:
   id: 152
   languages: Java
+Derby-jdbc-driver:
+  id: 153
+  languages: Java,Scala,Kotlin,Groovy
```

Comment:
All last three are Java as well. We don't need to repeat.

Reply:
ok